### DISCUSSION TITLE
Git Attributes not applied

### DISCUSSION THREAD

**Discussion by acquitelol** [community/community#142265] [Mobile]
In any repository, a feature on desktop is that you can set a linguist-language for specific file extensions in a .gitattributes file at the root which allows you to get syntax highlighting for files that would otherwise not have any. 

On mobile, however, these attributes are not applied at all (or very rarely). If you set any extension to be highlighted with another language's highlighting, it usually doesn't apply (however it does once every few hundred attempts at opening a file, which is very strange).

I'm not sure if this is the place to report bugs, but it's been an issue for months using the mobile app and nothing has ever been done about it.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.182.0 (144168013)
Device: iPhone 12 (iOS 16.3.1)
</details>

**Comment by taki-on** [comment]
Hi @acquitelol would you mind sharing a screenshot about the issue? We support syntax highlighting for some well-known languages, but we do not support configuring it via `.gitattributes`. However, I am curious about the issue that you sometimes get it syntax-highlighted. A screenshot would be very helpful to understand the issue. Thank you!

**Reply by acquitelol** [reply]
Sorry for taking so long I completely forgot about this issue.

https://github.com/user-attachments/assets/03532abf-f048-4a8e-a0f3-5704209a88d1

**Reply by acquitelol** [reply]
For context, this repository has a .gitattributes file at the root with `*.le linguist-language=Rust` inside. It's not on my iPhone but the same issue occurs there too.

**Comment by Ppoollkkmmnn09** [comment]
> Hi @acquitelol would you mind sharing a screenshot about the issue? We support syntax highlighting for some well-known languages, but we do not support configuring it via `.gitattributes`. However, I am curious about the issue that you sometimes get it syntax-highlighted. A screenshot would be very helpful to understand the issue. Thank you! 


Cant parse page
Can mercene me video how to fix it
![Screenshot_٢٠٢٥٠١٢١-١٩٣٦٥٦.jpg](https://github.com/user-attachments/assets/1188df96-cde0-44e8-bf6d-54458223580e)
